lang/cs/Org.Apache.REEF.Wake/Remote/Impl/StreamingLink.cs

```diff
@@ -23,9 +23,9 @@
 using System.Threading;
```

Comment:
This class still has references to `IWritable` in the comments. Those are outdated, correct?